postpic/datahandling.py

```diff
@@ -336,8 +361,10 @@ def extent(self, newextent):
         if not self.dimensions * 2 == len(newextent):
             raise TypeError('size of newextent doesnt match self.dimensions * 2')
         for i in range(len(self.axes)):
-            self.axes[i].setextent(newextent[2 * i:2 * i + 2],
-                                   self.matrix.shape[i])
+            newax = copy.copy(self.axes[i])
+            newax.setextent(newextent[2 * i:2 * i + 2],
```

Comment:
`Field` has a `extent.setter`, i.e. a writable property, while `Axis` has a `setextent` method. Have the same in both?